controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -669,23 +323,45 @@ func remove(slice []string, s string) (result []string) {
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *StorageConsumerReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	enqueueStorageConsumerRequest := handler.EnqueueRequestsFromMapFunc(
+		func(obj client.Object) []reconcile.Request {
+			labels := obj.GetLabels()
+			if value, ok := labels[StorageConsumerNameLabel]; !ok {
+				return []reconcile.Request{}
+			} else {
+				return []reconcile.Request{{
+					NamespacedName: types.NamespacedName{
+						Name:      value,
+						Namespace: obj.GetNamespace(),
+					},
+				}}
+			}
```

Comment:
```suggestion
			if value, ok := labels[StorageConsumerNameLabel]; ok {
				return []reconcile.Request{{
					NamespacedName: types.NamespacedName{
						Name:      value,
						Namespace: obj.GetNamespace(),
					},
				}}
			}
			return []reconcile.Request{}
```

golint is complaining. It can be updated like this.

Reply:
thanks, done